tests/goal_tests.cpp

```diff
@@ -1,131 +1,130 @@
 #include <pick_ik/frame.hpp>
```

Comment:
Can we make sure the very first #include is the header under test? frame.hpp is already transitively included anyways so we don't need this here.